spec/models/cti/caller_id_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe User do
```

Comment:
I removed Phony because the only method used was `Phony.plausible?` and I could satisfy all test cases without Phony. `Phony.split` could be useful to display the number in a readable format.